meinberlin/assets/scss/style_user_facing.scss

```diff
@@ -50,7 +50,7 @@
 @import "../berlin_css/berlin_marketing";
 
 // extra scss / overwrites
-@import "../extra_css/swiper.scss";
+@import "../extra_css/swiper";
```

Comment:
removed as didn't pass linter, not sure why there wasn't an error when commiting, maybe we need to check out commit linting?